src/pages/dev/reference/glossary.mdx

```diff
@@ -0,0 +1,53 @@
+# Axelar Glossary
+
+The purpose of this document is to provide you with a better understanding of the terminology used in the Axelar documentation. 
+
+## Canonical
```

Comment:
If you are making this glossary global, then you need to add more context to these terms, as they only make sense in the context of the Interchain Token Service

Reply:
Updated

Reply:
Canonical does not mean this universally. You need to say "Canonical Token" or "Canonical Interchain Token"